packages/appium/docs/en/contributing.md

```diff
@@ -0,0 +1,123 @@
+---
+hide:
+  - navigation
+
+title: Contributing to Appium
+---
+
+The Appium project would not exist without the many contributions of code, documentation,
+maintenance, and support from companies and volunteers. As such, we welcome contributions!
+
+There are a lot of different ways to help the project - see below for everything you can do and the
```

Comment:
missing comma before `and`

Reply:
The two parts of the sentence are linked (`see below for everything` and `see below for [..] the processes`), so I don't think the comma is required

Reply:
I'm not an expert in English grammar, but we are lucky to have one (cc @jlipps)

Reply:
Comma is optional in this case. It creates a pause for the reader that separates it from `and the processes`, but is not grammatically required. It's just a question of style in this case really.